iroh-net/src/magicsock.rs

```diff
@@ -388,11 +390,10 @@ impl Inner {
                     }
                 }
 
-                let n = transmits.len();
-
                 // send derp
                 if let Some(ref derp_url) = derp_url {
                     self.try_send_derp(derp_url, public_key, split_packets(&transmits));
+                    transmits_sent = transmits.len();
```

Comment:
this seems to be mostly a wish to be fair, given how `try_send_derp` can drop things on the floor if it feels like it.  but for the purpose of this fix it's probably fine to leave as it.

Reply:
i guess that's a fair way to look at it